client/src/components/auth/RegistrationForm.js

```diff
@@ -11,139 +11,153 @@ import { createTeam } from '../../actions/team';
 import './Register.css';
 import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import toast from 'react-hot-toast';
+import { Formik } from 'formik';
+import * as yup from 'yup';
 
 export default function RegistrationForm() {
   const dispatch = useDispatch();
   const history = useHistory();
-  const [validated, setValidated] = useState(false);
   const isRegistered = useSelector(state => state.auth.isRegistered)
 
   useEffect(() => {
-    if(isRegistered && validated){
+    if(isRegistered){
       history.push('/login')
     }
-  }, [history, isRegistered, validated])
+  }, [history, isRegistered])
 
-  const [inputs, setInputs] = useState({
+  const teamInfoSchema = yup.object({
+    teamName: yup
+      .string()
+      .required('Team Name is required')
+      .min(3, 'Must be at least 3 characters'),
+    orgName: yup
+      .string()
+      .required('Organization Name is required')
+      .min(3, 'Must be at least 3 characters'),
+    email: yup
+      .string()
+      .email('Invalid Email')
+      .required('Email is required'),
+    password: yup
+      .string()
+      .required('Please Enter your password')
+      .matches(
+        /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
+        "Use 8 or more characters with a mix of letters, numbers & symbols"),
```

Comment:
This message can be improved to be more specific about the password policy, that you need at least 1 digit and symbol on top of the characters.